src/bun.js/webcore/body.zig

```diff
@@ -158,6 +158,18 @@ pub const Body = struct {
 
             return false;
         }
+
+        pub fn isDisturbed2(this: *const PendingValue, globalObject: *JSC.JSGlobalObject) bool {
```

Comment:
why can't we use isDisturbed?

Reply:
Because can or not be an Request instance or an unknow instance, we only care about the readable stream.